src/sdk/pynni/tests/test_tuner.py

```diff
@@ -17,107 +17,181 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT
 # OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 # ==================================================================================================
-
-
-import nni.protocol
-from nni.protocol import CommandType, send, receive
-from nni.tuner import Tuner
-from nni.msg_dispatcher import MsgDispatcher
-from nni.utils import extract_scalar_reward
-from io import BytesIO
+import glob
 import json
+import logging
+import os
+import shutil
+import sys
 from unittest import TestCase, main
 
+from nni.batch_tuner.batch_tuner import BatchTuner
+from nni.evolution_tuner.evolution_tuner import EvolutionTuner
+from nni.gp_tuner.gp_tuner import GPTuner
+from nni.gridsearch_tuner.gridsearch_tuner import GridSearchTuner
+from nni.hyperopt_tuner.hyperopt_tuner import HyperoptTuner
+from nni.metis_tuner.metis_tuner import MetisTuner
+from nni.smac_tuner.smac_tuner import SMACTuner
```

Comment:
There is a problem on windows: https://msrasrg.visualstudio.com/NNIOpenSource/_build/results?buildId=6298
```
======================================================================
ERROR: test_tuner (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_tuner
Traceback (most recent call last):
  File "C:\agent\_work\_tool\Python\3.6.4\x64\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\agent\_work\_tool\Python\3.6.4\x64\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "C:\agent\_work\5\s\src\sdk\pynni\tests\test_tuner.py", line 34, in <module>
    from nni.smac_tuner.smac_tuner import SMACTuner
  File "C:\agent\_work\5\s\src\sdk\pynni\nni\smac_tuner\__init__.py", line 1, in <module>
    from .smac_tuner import SMACTuner
  File "C:\agent\_work\5\s\src\sdk\pynni\nni\smac_tuner\smac_tuner.py", line 29, in <module>
    from smac.facade.epils_facade import EPILS
ModuleNotFoundError: No module named 'smac'
----------------------------------------------------------------------
Ran 28 tests in 6.807s
FAILED
 (errors=1)
```
Since there is problem on pipeline itself, the status of the build is sucessful,  it should be failed.